### DISCUSSION TITLE
Nothing downloads on M1 Mac

### DISCUSSION THREAD

**Discussion by DudleyAH** [sircharlo/meeting-media-manager#53] [Q&A] [answered]
Hi, Thanks for working on this project. I have just installed on M1 Mac, loads in OK, but after loading in, all I get is the circle going round and round and nothing else appears. Is this something specific to M1 Mac. Also tried the Windows 10 version from a Parallels VM inside the M1 Mac (The ARM version of windows), but same result. :-(
![Screenshot 2021-06-05 at 20 24 44](https://user-images.githubusercontent.com/24960785/120903096-1524c200-c63c-11eb-810e-9351006797b8.png)
Any ideas what the fix may be? 
Thanks for any advice
Rob

**Comment by sircharlo** [comment]
Hm interesting. I don’t have a Mac, much less an M1 one, so apologies for this bug. Could you hit Command+Option+J and tell me if there are any errors please?

**Reply by sircharlo** [reply]
In any case, I just pushed an update that might help. Try it [here](https://github.com/sircharlo/jw-meeting-media-fetcher/releases/latest).

**Comment by DudleyAH** [comment]
Thanks Charles

Will give it an try and get back to you😊

**Comment by DudleyAH** [comment]
Hi Charles

Just tried the updated version, installed fine as a MAC universal app, but still doesn’t get past the same screen, with just the circle going Rond and no further :-( 

Thank you Rob

**Reply by sircharlo** [reply]
Ok, same goes then: hit Command+Option+J (or maybe Command+Option+I) on that screen and tell me what shows up in the Console tab

**Comment by DudleyAH** [comment]
Hi Charles

Command Option J didn’t bring up anything, but when I enabled deliver tools, could see this error in the console

**Reply by sircharlo** [reply]
Sorry, I can't see anything here. Maybe try copy-pasting the error text?

**Comment by DudleyAH** [comment, marked as answer]
This is the error that shows in the console


Date locale en not found, falling back to "en"
media-updater.js:816 Uncaught (in promise) TypeError: Cannot read property 'symbol' of undefined
    at getTranslations (media-updater.js:816)
    at getInitialData (media-updater.js:687)
﻿

**Reply by sircharlo** [reply]
Wow that's a glaring bug. Embarrassing, really. Pushing a fix in about 15 minutes, hopefully that will be the last bug for you 😆

**Reply by sircharlo** [reply]
It's live now.

**Comment by DudleyAH** [comment]
Thanks will try shortly 😁

**Comment by DudleyAH** [comment]
Hi Charles

Yes, thats’ fixed, working great now, thanks for the speedy issue resolve :-)

Rob

**Reply by sircharlo** [reply]
Awesome! Glad to hear it :) thanks for bringing it to my attention.

**Comment by DudleyAH** [comment]
If you ever want me to test out any other releases on M1 Mac just let me know 😀, thanks again though for the fix.

**Reply by sircharlo** [reply]
Will do for sure. I’m quite frankly surprised it works..!

**Comment by DudleyAH** [comment]
Works well now, even the mp4 conversion worked as expected, so well done !
